nsdb-cluster/src/main/scala/io/radicalbit/nsdb/cluster/coordinator/ReadCoordinator.scala

```diff
@@ -241,6 +224,22 @@ class ReadCoordinator(metadataCoordinator: ActorRef,
                     tags = retrieveField(bits, distinctField, (bit: Bit) => bit.tags)
                   )
                 }.map(limitAndOrder(_, statement, schema))
+
+              case Right(ParsedGlobalAggregatedQuery(_, _, _, limit, fields, _, _)) =>
+                gatherNodeResults(statement, schema, uniqueLocationsByNode) { seq =>
+                  val count = min(limit, seq.map(_.value.rawValue.asInstanceOf[Long]).sum)
+                  if (fields.nonEmpty) {
+                    applyOrderingWithLimit(
+                      seq.map { bit =>
+                        bit.copy(tags = bit.tags + ("count(*)" -> count))
```

Comment:
string might be a constant?

Reply:
Yes, there are terms that deserve to be treated as constants. e.g. 
```
count(*), *, value
```

I wouldn't like to further load this PR, but I understand that at some point we must manage it.

Reply:
👍